src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = str => Number(str.replace(/[^\d.-]/g, ''));
+
+const sortList = (list, sortParam) =>
+  [...list].sort(
+    (a, b) =>
+      convertToNumber(b.dataset[sortParam]) -
+      convertToNumber(a.dataset[sortParam]),
+  );
+
+const getEmployees = (sortedList, element) => {
+  element.innerHTML = '';
+
+  sortedList.forEach(item => {
+    const li = document.createElement('li');
+
+    li.setAttribute('data-position', item.dataset.position);
+    li.setAttribute('data-salary', item.dataset.salary);
+    li.setAttribute('data-age', item.dataset.age);
+
+    li.textContent = item.innerText;
+
+    element.appendChild(li);
+  });
+};
```

Comment:
The `getEmployees` function seems to be recreating the list of employees in the provided element. This could be inefficient if the list is large. Instead, consider manipulating the order of the existing list items in the DOM. This could be done in the `sortList` function, which could return the sorted list of elements that can then be appended to the list in the DOM.